{
	"id": "CVE-2023-26150",
	"references": [
		{
			"type": "exploit",
			"source": "NVD",
			"url": "https://security.snyk.io/vuln/SNYK-PYTHON-ASYNCUA-5673435",
			"time_stamp": "2023-10-04T17:12:00Z"
		},
		{
			"type": "exploit",
			"source": "NVD",
			"url": "https://gist.github.com/artfire52/84f7279a4119d6f90381ac49d7121121",
			"time_stamp": "2023-10-04T17:12:00Z"
		},
		{
			"type": "exploit",
			"source": "NVD",
			"url": "https://github.com/FreeOpcUa/opcua-asyncio/issues/1014",
			"time_stamp": "2023-10-04T17:12:00Z"
		}
	]
}
